DQMOffline/Trigger/python/B2GMonitoring_cff.py

```diff
@@ -1,4 +1,45 @@
 import FWCore.ParameterSet.Config as cms
 
+from DQMOffline.Trigger.JetMETMonitor_cfi import hltJetMETmonitoring
+from DQMOffline.Trigger.HTMonitor_cfi import hltHTmonitoring
+
+# HLT_AK8PFHT700_TrimR0p1PT0p03Mass50                                                                                                                                         
+AK8PFHT700_HTmonitoring = hltHTmonitoring.clone()
+AK8PFHT700_HTmonitoring.FolderName = cms.string('HLT/B2GMonitor/AK8PFHT700_TrimR0p1PT0p03Mass50')
+AK8PFHT700_HTmonitoring.numGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_AK8PFHT700_TrimR0p1PT0p03Mass50_v*")
```

Comment:
can you please, point to the JIRA ?
(I do not see this path in the silvio google doc)